src/qt/menu.cpp

```diff
@@ -56,23 +57,54 @@ static void InsertMenuItemAction( const wxMenu *menu, const wxMenuItem *previous
 {
     QMenu *qtMenu = menu->GetHandle();
     QAction *itemAction = item->GetHandle();
-    if ( item->GetKind() == wxITEM_RADIO )
+    switch ( item->GetKind() )
     {
-        // If the previous menu item is a radio item then add this item to the
-        // same action group, otherwise start a new group:
-
-        if ( previousItem != NULL && previousItem->GetKind() == wxITEM_RADIO )
-        {
-            QAction *previousItemAction = previousItem->GetHandle();
-            QActionGroup *previousItemActionGroup = previousItemAction->actionGroup();
-            wxASSERT_MSG( previousItemActionGroup != NULL, "An action group should have been setup" );
-            previousItemActionGroup->addAction( itemAction );
-        }
-        else
+        case wxITEM_RADIO:
+            // If the previous menu item is a radio item then add this item to the
+            // same action group, otherwise start a new group:
+
+            if ( previousItem != NULL && previousItem->GetKind() == wxITEM_RADIO )
+            {
+                QAction *previousItemAction = previousItem->GetHandle();
+                QActionGroup *previousItemActionGroup = previousItemAction->actionGroup();
+                wxASSERT_MSG( previousItemActionGroup != NULL, "An action group should have been setup" );
+                previousItemActionGroup->addAction( itemAction );
+            }
+            else
+            {
+                QActionGroup *actionGroup = new QActionGroup( qtMenu );
+                actionGroup->addAction( itemAction );
+                wxASSERT_MSG( itemAction->actionGroup() == actionGroup, "Must be the same action group" );
+            }
+            break;
+        case wxITEM_NORMAL:
         {
-            QActionGroup *actionGroup = new QActionGroup( qtMenu );
-            actionGroup->addAction( itemAction );
-            wxASSERT_MSG( itemAction->actionGroup() == actionGroup, "Must be the same action group" );
+            wxWindowID id = item->GetId();
+            if ( wxIsStockID( id ) )
+            {
+                itemAction->setText( wxQtConvertString( wxGetStockLabel( id ) ) );
+                wxAcceleratorEntry accel = wxGetStockAccelerator( id );
+                QString opcode(QStringLiteral("Ctrl+"));
```

Comment:
This assumes that all accelerators use `Ctrl` and while this is indeed currently the case, I think it would be cleaner to still add `if ( accel.GetFlags() & wxACCEL_CTRL )` test here for the peace of mind.